operator/config/overlays/development/kustomization.yaml

```diff
@@ -17,6 +17,14 @@ labels:
     app.kubernetes.io/instance: loki-operator-v0.0.1
     app.kubernetes.io/version: "0.0.1"
 
+generatorOptions:
+  disableNameSuffixHash: true
```

Comment:
Any reason for disabling this? Apart from generating static configurations I usually keep the hash enabled so that, for example, deployments automatically update to the newest generated ConfigMap/Secret.

Reply:
This comes from kubebuilder scaffolding TBH, so I kept it w/o asking.